src/mlpack/tests/split_data_test.cpp

```diff
@@ -94,8 +94,7 @@ void CheckDuplication(const Row<size_t>& trainLabels,
 TEST_CASE("SplitShuffleDataResultMat", "[SplitDataTest]")
 {
   mat input(2, 10);
-  size_t count = 0; // Counter for unique sequential values.
-  input.imbue([&count] () { return ++count; });
+  input = linspace(0, input.n_elem - 1);
```

Comment:
Don't you need to specify `N` here? https://arma.sourceforge.net/docs.html#linspace

linspace gives a vector though, so if you want this to be a matrix, maybe you need to do something like... `mat input = reshape(linspace<mat>(0, 19, 20), 2, 10);` (you should double-check that, I am not 100% sure)

Reply:
Let us if the tests pass :smiley: